[EdgeTPU] Extract Interface for Modularization (CfgData)

Hyeon-Uk opened the pull request:
This commit modularizes the CfgData file for EdgeTPU.

It defines the interface named ICfgData extracted from CfgData.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

related : #1621

rhqjatn2398 committed:
[EdgeTPU] Modularize CfgData for EdgeTPU

This commit modularize the CfgData file for EdgeTPU.

It defines the interface named ICfgData extracted from CfgData.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

Co-authored-by: hohee-hee <khappy517@gmail.com>
Co-authored-by: Hyeon-Uk <rlagusdnr120@gmail.com>

dayo09 commented on src/CfgEditor/ICfgData.ts:
```diff
@@ -0,0 +1,30 @@
+    updateSectionWithValue(section: string, value: string): void;
+    isSame(textStringified: string): boolean;
+    sort(): void;
+}
```
Could you add a new line at the end of the file?

Hyeon-Uk replied:
I've added a new line at the end of the file :)

Hyeon-Uk replied:
I've added a new line at the end of the file :)

Hyeon-Uk committed:
Add a new line in the end of ICfgData.ts

Adding a new line to comply with the format rule

ONE-vscode-DCO-1.0-Signed-off-by: Hyeon-Uk <rlagusdnr120@gmail.com>

Hyeon-Uk committed:
format ICfgData.ts file

to formatting

ONE-vscode-DCO-1.0-Signed-off-by: Hyeon-Uk <rlagusdnr120@gmail.com>

dayo09 reviewed (approved):
Thanks :-D

hohee-hee reviewed (approved):
LGTM :)

rhqjatn2398 reviewed (approved):
LGTM 👍 👍 💯
